src/arrayMethodSort.js

```diff
@@ -5,7 +5,27 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
let's set a default value for compare function as it will make a code shorter 
`[].__proto__.sort2 = function(`
 ` compareFunction = (a, b) => String(a) > String(b) ? 1 : -1`
 `) {`